apps/researcher/src/app/[locale]/communities/[slug]/edit-community-form.tsx

```diff
@@ -16,25 +17,22 @@ interface Props {
   slug: string;
   description?: string;
   attributionId?: string;
+  licence?: string;
+  licenceToAccept: string;
 }
 
 interface FormValues {
   name: string;
-  slug: string;
   description: string;
   attributionId: string;
+  licenceChecked: boolean;
```

Comment:
Does a user 'check' a license? Or 'accept' or 'agree to' a license?

Reply:
I have renamed the property to `agreedToLicense`